tests/test_gpu_aamp.py

```diff
@@ -45,7 +45,7 @@ def test_gpu_aamp_self_join(T_A, T_B):
     m = 3
     zone = int(np.ceil(m / 4))
     for p in [1.0, 2.0, 3.0]:
-        ref_mp = naive.aamp(T_B, m, exclusion_zone=zone, p=p)
+        ref_mp = naive.aamp(T_B, m, exclusion_zone=zone, p=p, row_wise=True)
```

Comment:
we now traverse row by row. (we are now consistent with `test_gpu_stump_self_join`)

Reply:
Perfect!